why-apollo-advantages-disadvantages-alternatives.md

```diff
@@ -23,25 +23,25 @@ summary = "It is not always simple to pick the right solution for a problem. In
 
 {{% chapter_header "Apollo Advantages" "apollo-advantages" %}}
 
-The following topics show you the major advantages of using Apollo.
+Major advantages of using Apollo.
 
 {{% sub_chapter_header "Apollo's Ecosystem" "apollo-ecosystem" %}}
 
-The Apollo ecosystem offers plenty of solutions for many problems even though GraphQL is in its early stages. And the ecosystem is growing. At every other conference the company is announcing an update for Apollo or another library which can be used complementary to Apollo's tech stack. For instance, that's why Apollo is not only covering GraphQL but also REST interfaces for enabling backward compatibility to RESTful architectures. It goes beyond GraphQL being used only for the network layer and remote data by offering a state management solution for local data too. So if you make the leap to use Apollo for your application, you are welcomed with a great ecosystem.
+While GraphQL is in its early stages, the Apollo ecosystem offers solutions for many of its challenges.  Beyond that we can see how much the ecosystem is growing, because the company announces an update for Apollo or another library that can be used with Apollo's tech stack at every other technology conference.  Apollo isn't just covering GraphQL, though; they also have effort invested in REST interfaces for backward compatibility to RESTful architectures.  This takes GraphQL beyond the network layer and remote data by offering a state management solution for local data, too. 
 
 {{% sub_chapter_header "The Company and Community behind Apollo" "apollo-company-community" %}}
 
-The company behind Apollo (Meteor Development Group Inc.) is pouring lots of resources into its endeavour. They are active in open source, write lots of articles about their product, and are seen often at conferences. In general, the GraphQL ecosystem {{% a_blank "seems to be in good shape for the future" "https://techcrunch.com/2018/05/15/prisma" %}}. Many people are speaking about it and are keen to adopt it for their own applications. In addition, even though Apollo is powered by its company, the community behind it is growing. More people are adopting GraphQL and opt-in Apollo for their client-side and server-side JavaScript applications.
+The company behind Apollo--Meteor Development Group Inc.--is pouring lots of resources into its success. Meteor is also active in open source, offering in-depth articles about their products, supported by an established presence at the conferences. In general, the GraphQL ecosystem {{% a_blank "seems to be in good shape for the future" "https://techcrunch.com/2018/05/15/prisma" %}}. The community behind Graph QL is growing, as more developers are adopt it and use Apollo for client and server-side JavaScript applications.
```

Comment:
"The company behind Apollo--Meteor Development Group Inc.--is pouring lots" -> "The company behind Apollo is pouring lots"

Reply:
Why would you omit the name of the company? This is useful information to anyone who wants to know about Apollo?

Reply:
I think mentioning Apollo's company here might be misleading, because Meteor is a popular framework that they have invented before (hence the name), and I wouldn't want to explain it in this chapter. In addition, maybe something about the company changes but this is not news that I would actively receive somehow. So it may be just easier for everyone to leave it out, because it doesn't add much value. I think no one would look up the company.